datasource.go

```diff
@@ -238,6 +241,8 @@ func (ds *SQLDatasource) handleQuery(ctx context.Context, req backend.DataQuery,
 		return res, nil
 	}
 
+	// err = Unwrap(err)
```

Comment:
@scottlepp Shall we remove this if it's not needed?

Reply:
yep.  the [error from sdk now implements this ](https://github.com/grafana/grafana-plugin-sdk-go/commit/ece8a549fad9e723ce7c882f1c789ba09e286934#diff-c8540b7c029d56badd24565f8bf64410afb8ea24f76bc0f9e6e21322ebc59cffR24) so we don't need this

Reply:
I mean removing instead of commenting :)

Reply:
yep.  i missed it before merging